bodhi/tests/server/base.py

```diff
@@ -52,15 +48,13 @@
 class BaseTestCase(unittest.TestCase):
     def setUp(self):
         bugs.set_bugtracker()
-        engine = create_engine(DB_PATH)
-        # We want a special session that lasts longer than a transaction
-        self.Session = scoped_session(
-            sessionmaker(bind=engine, extension=ZopeTransactionExtension(keep_session=True)))
-        log.debug('Creating all models for %s' % engine)
-        models.Base.metadata.bind = engine
-        models.Base.metadata.create_all(engine)
+        self.config = {'sqlalchemy.url': DB_PATH}
+        self.engine = initialize_db(self.config)
+        self.Session = Session
```

Comment:
If ```self.Session = bodhi.server.Session``` now, maybe we can drop this line and adjust tests that were using ```self.Session```. Actually, it may be that none or few tests do use this - I may have only added this line so I could call ```self.Session.remove()``` in ```tearDown()```.